docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2.1'
+
+services:
+  postgres:
+    user: postgres
+    image: postgres
```

Comment:
I suggest to specify a version here.

Reply:
not sure about that, this way we use the latest one, @beniwohli would you like to have a fixed version for the services?

Reply:
I think it's fine to have the latest version. Generally speaking, our interaction with Postgres is so limited, that I don't think any version change would impact our tests